src/main/java/org/breedinginsight/services/validators/TraitValidatorError.java

```diff
@@ -66,6 +66,10 @@ public ValidationError getMissingScaleDataTypeMsg() {
     }
 
     @Override
+    public ValidationError getPeriodObsVarNameMsg() {
+        return new ValidationError("observationVariableName", "Period in name is invalid", HttpStatus.BAD_REQUEST);
+    }
```

Comment:
I think it would be good to keep the error messages consistent between TraitValidatortError and TraitFileValidatorError, so this message could be changed to "Period is invalid".

Reply:
This is a banner error message.  Table messages have a field the tells the user which field contains the error, banner messages do not.  So, it was decided that this message should contain the name of the filed.